src/Aquifer.API/Aquifer.API.csproj

```diff
@@ -2,11 +2,6 @@
 
     <PropertyGroup>
         <TargetFramework>net8.0</TargetFramework>
-        <Nullable>enable</Nullable>
```

Comment:
Will we still get IDE warnings with this not here any more? Like sometimes I'll get the warning from the IDE before build that something could be null that isn't allowed to be.

Reply:
Yes, we will still get the warnings because this project still defines `<Nullable>enable</Nullable>`.  All projects in the solution now inherit properties defined in `Directory.Build.Props`, this setting is defined [there](https://github.com/BiblioNexusStudio/aquifer-server/pull/499/files#diff-9da24614831c308827a1ae533ffea392c97638c261dd42bd0f5226baa136d16eR5), and thus it is redundant to continue including the setting at the project level (unless we want to override the solution defaults).  And now any new projects in the solution will get this setting for free and won't have to remember to include this line.

Reply:
Okay, I just wanted to make sure that other file (since it has Build in the name), wasn't only applying on build.